scripts/upgrade_version.py

```diff
@@ -0,0 +1,85 @@
+import click
```

Comment:
please move this under `ci/ray_ci` directory, and make it a bazel `py_binary`, and make it the official way to run it with `bazelisk run`

Reply:
fixed!